python/cudf/cudf/io/parquet.py

```diff
@@ -872,6 +955,13 @@ class ParquetDatasetWriter:
     file_name_prefix : str
         This is a prefix to file names generated only when
         `max_file_size` is specified.
+    storage_options : dict, optional, default None
+        Extra options that make sense for a particular storage connection,
+        e.g. host, port, username, password, etc. For HTTP(S) URLs the
+        key-value pairs are forwarded to ``urllib.request.Request`` as
+        header options. For other URLs (e.g. starting with “s3://”, and
+        “gcs://”) the key-value pairs are forwarded to ``fsspec.open``.
```

Comment:
Careful when copying from rendered docs online. Sphinx will generate smart quotes from regular quotes by default, which shows up on the online docs, but we don't want to use smart quotes in our source code.
```suggestion
        header options. For other URLs (e.g. starting with "s3://", and
        "gcs://") the key-value pairs are forwarded to ``fsspec.open``.
```

Reply:
Yikes